index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="de">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lichtblick-Rebuild</title>
+    <style class="style-heading">
+      @import url("https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap");
+    </style>
+    <style class="style-other-text">
+      @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap");
+    </style>
```

Comment:
Der Inhalt der beiden `<style>`-Tags solltest du am Besten in deine CSS-Datei verschieben; auf die Weise werden die Schriften in deiner Style-Datei geladen. Ansonsten müsstest du sie für jede HTML-Datei neu einfügen.